### ISSUE TITLE
about the loss function in training

### ISSUE THREAD

**Issue by TonyZ-sz** [siriusdemon/pytorch-DEX#7]
hi, sorry to bother. I wanna use your written model to train my own datasets. noticed that the expected_age is calculated directly by the output of model. when doing training, is the loss function cross-entropy between the output of model and the sequence where 1 at the age and 0 otherwise?
